src/scores/categorical/contingency_impl.py

```diff
@@ -551,96 +554,153 @@ def true_negative_rate(self) -> xr.DataArray:
         """
         return self.probability_of_detection()
 
-    def precision(self):
+    def precision(self) -> xr.DataArray:
         """
         Identical to the Success Ratio.
 
-        https://en.wikipedia.org/wiki/Precision_and_recall
+        What proportion of the forecast events actually eventuated?
+
+        Returns:
+            xr.DataArray: An xarray object containing the precision score
+
+        .. math::
+            \\text{precision} = \\frac{\\text{true positives}}{\\text{true positives} + \\text{false positives}}
+
+        Notes:
+            - Range: 0 to 1.  Perfect score: 1.
+            - "True positives" is the same as "hits"
+            - "False positives" is the same as "misses"
+
+        References:
+            - https://www.cawcr.gov.au/projects/verification/#SR
+            - https://en.wikipedia.org/wiki/Precision_and_recall
+
         """
         return self.success_ratio()
 
-    def f1_score(self):
+    def f1_score(self) -> xr.DataArray:
         """
         Calculates the F1 score.
-        https://en.wikipedia.org/wiki/F-score
+
+        Returns:
+            xr.DataArray: An xarray object containing the F1 score
+
+        .. math::
+            \\text{F1} = \\frac{2 \\times \\text{true positives}}{(2 \\times  \\text{true positives}) + \\text{false positives} + \\text{false negatives}}
+
+        Notes:
+            - "True positives" is the same as "hits".
+            - "False positives" is the same as "false alarms".
+            - "False negatives" is the same as "misses".
+
+        References:
+            - https://en.wikipedia.org/wiki/F-score
         """
         cd = self.counts
         f1 = 2 * cd["tp_count"] / (2 * cd["tp_count"] + cd["fp_count"] + cd["fn_count"])
         return f1
 
-    def equitable_threat_score(self):
+    def equitable_threat_score(self) -> xr.DataArray:
         """
-        Calculates the Equitable threat score (also known as the Gilbert skill score).
+        Identical to the Gilbert Skill Score.
+
+        Calculates the Equitable threat score.
 
         How well did the forecast "yes" events correspond to the observed "yes"
         events (accounting for hits due to chance)?
 
-        Range: -1/3 to 1, 0 indicates no skill. Perfect score: 1.
-
+        Returns:
+            xr.DataArray: An xarray object containing the equitable threat score
 
-        References:
+        .. math::
+            \\text{ETS} = \\frac{\\text{hits} - \\text{hits} _\\text{random}}\
+            {\\text{hits} + \\text{misses} + \\text{false alarms} - \\text{hits} _\\text{random}}
 
-        Gilbert, G.K., 1884. Finley’s tornado predictions. American Meteorological Journal, 1(5), pp.166–172.
+        Notes:
+            - Range: -1/3 to 1, 0 indicates no skill. Perfect score: 1.
+            - "Hits" is the same as "true positives".
+            - "Misses" is the same as "false negatives".
+            - "False alarms" is the same as "false positives".
```

Comment:
Swap order of "hits" and "true positives" etc.  (for consistency)

Reply:
Fixed, thanks